Solutions/PDNS Block Data Connector/DataConnectors/PDNSBlockDataConnector_API_FunctionApp.json

```diff
@@ -0,0 +1,122 @@
+    "graphQueries": [
+		{
+			"metricName": "Total data received",
+			"legend": "PDNSBlockData_CL",
+			"baseQuery": "PDNSBlockData_CL"
+    	}
+    ],
+    "sampleQueries": [
+		{
+			"description": "All PDNS Block Data",
+			"query": "PDNSBlockData_CL\n | sort by TimeGenerated desc"
+    	}
+    ],
+    "dataTypes": [
+		{
+        	"name": "PDNSBlockData_CL",
+        	"lastDataReceivedQuery": "PDNSBlockData_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+    	}
+    ],
+    "connectivityCriterias": [
+		{
+			"type": "IsConnectedQuery",
+			"value": [
+				"PDNSBlockData_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+        	]
+    	}
+	],
+	"availability": {
+		"status": 1,
+		"isPreview": false
+	},
+	"permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            },
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+                "providerDisplayName": "Keys",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "action": true
+                }
+            }
+        ],
+		"customs": [
+			{
+				"name": "Microsoft.Web/sites permissions",
+				"description": "Read and write permissions to Azure Functions to create a Function App is required. [See the documentation to learn more about Azure Functions](https://docs.microsoft.com/azure/azure-functions/)."
+			},
+			{
+				"name": "REST API Credentials/permissions",
+				"description": "**AWSAccessKeyId** and **AWSSecretAccessKey** are required for making AWS API calls."
+			}
+		]
+	},
+	"instructionSteps": [
+		{
+			"title": "",
+			"description": ">**NOTE:** This connector uses Azure Functions to pull logs into Microsoft Sentinel. This might result in additional costs for data ingestion. Check the [Azure Functions pricing page](https://azure.microsoft.com/pricing/details/functions/) for details."
+		},
+		{
+			"title": "",
+			"description": "**STEP 1 - To configure access to the S3 Bucket containing your PDNS Data Blocks, use the Access Key ID and Secret Access Key that were provided to you."

```

Comment:
Hi @TedWhi please maintain consistency in having text style as bold in the instruction steps in case of step 1 it is visible in the following format

![image](https://github.com/Azure/Azure-Sentinel/assets/115772287/3145a667-ec70-4f14-9dac-8539f5a2ceaa)